dev/ci/user-overlays/README.md

```diff
@@ -64,12 +64,12 @@ named `more_efficient_tc`.
 - Make necessary changes, then rerun the script to verify they work.
 - From the `<job name>` subdirectory, commit your changes to a new
   branch, based on the base branch name listed in `ci-basic-overlay.sh`,
-  for example `coq-master` for elpi.
+  for example `master` for elpi.
 - If necessary, fork the external project from the project's github page.
   (Only needs to be done once, ever.)
 - Push to the external project and create a new PR.  Make sure you pick
   the correct base branch in the github GUI for the comparison
-  (e.g. `coq-master` for elpi).
+  (e.g. `master` for elpi).
```

Comment:
````suggestion
  (e.g. `master` for elpi).
  - Beware some projects (e.g. elpi) use `ppx_optcomp` to make the same branch compile
    with multiple version of Coq. See https://github.com/janestreet/ppx_optcomp for the doc.
    Short example:
 ```diff
--- a/src/coq_elpi_builtins.ml
+++ b/src/coq_elpi_builtins.ml
@@ -518,12 +518,20 @@ let set_accumulate_to_db_interp, get_accumulate_to_db_interp =
   (fun x -> f := x),
   (fun () -> !f)
 
+[%%if coq = "8.19" || coq = "8.20"]
 let is_global_level env u =
   try
     let set = Univ.Level.Set.singleton u in
     let () = UGraph.check_declared_universes (Environ.universes env) set in
     true
   with UGraph.UndeclaredLevel _ -> false
+[%%else]
+let is_global_level env u =
+  let set = Univ.Level.Set.singleton u in
+  match UGraph.check_declared_universes (Environ.universes env) set with
+  | Ok () -> true
+  | Error _ -> false
+[%%endif]
 
 let err_if_contains_alg_univ ~depth t =
   let global_univs = UGraph.domain (Environ.universes (Global.env ())) in
 ```
````